source/SpinalHDL/Semantic/when_switch.rst

```diff
@@ -97,6 +97,65 @@ is equivalent to
     }
   }
 
+
+Additional options
+^^^^^^^^^^^^^^^^^^
+
+Sometimes handling all cases can become unwieldy and error prone so SpinalHDL, by default, will handle the uncovered cases with the last ``is`` block.
```

Comment:
```suggestion
Sometimes handling all cases can become unwieldy and error prone so SpinalHDL, by default, handles the uncovered cases with the last ``is`` block.
```

It is implemented and not to be implemented, so I think using present of general truths here is clearer.